website/docusaurus.config.js

```diff
@@ -313,6 +313,9 @@ ${content}`,
       algoliaOptions: {}, // Optional, if provided by Algolia
       debug: false, // Set debug to true if you want to inspect the modal
     },
+    customFields: {
+      blockNativeApiKey: process.env.BLOCK_NATIVE_API_KEY || "DummyKey",
```

Comment:
Just curious, is the `DummyKey` meant for something when the API key isn't provided? Like for a test or to silence an error, etc?

Reply:
so that it wont be an undefined as the api key.